Sources/MapboxDirections/Incident.swift

```diff
@@ -21,46 +20,174 @@ public struct Incident: Codable, Equatable {
         case geometryIndexStart = "geometry_index_start"
         case geometryIndexEnd = "geometry_index_end"
     }
-
-    var identifier: String
-    var type: String
-    var description: String
-    var creationTime: String
-    var startTime: String
-    var endTime: String
-    var impact: String?
-    var subtype: String?
-    var subtypeDescription: String?
-    var alertCodes: [Int]
-    var lanesBlocked: [Int]
-    var geometryIndexStart: Int
-    var geometryIndexEnd: Int
+    
+    /// Defines known types of incidents.
+    ///
+    /// Each incident may or may not have specific set of data, depending on it's `kind`
+    public enum Kind: String {
+        case Accident = "accident"
+        case Congestion = "congestion"
+        case Construction = "construction"
+        case DisabledVehicle = "disabled_vehicle"
```

Comment:
jazzy is currently configured not to show any undocumented symbol, so although these kind enumerations will show up, none of their cases will. It’s a lot to document, so if you need to just document each one pro forma (such as “Disabled vehicle” here), that’s fine for now.